dist/assests/css/style.css

```diff
@@ -78,11 +78,44 @@ ul {
   text-align: center;
 }
 
-/* Promotions Section */
-.promotions-section__wrapper {
-  display: grid;
-  grid-template-columns: repeat(2, 1fr);
-  text-align: center;
+/* Promotions and Contact Sections */
+.promotion-contact-flexwrapper {
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.promotion-contact-flexwrapper section {
```

Comment:
try to avoid element selectors in CSS whenever possible, CSS classes should be your main way of styling.
recommended reads:
https://css-tricks.com/strategies-keeping-css-specificity-low/
https://frontstuff.io/you-need-to-stop-targeting-tags-in-css

Reply:
Got it! will fix it right away.